docs/resources/property_set.md

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Configuration"
+---
+# Artifactory Property Set Resource
+
+Provides an Artifactory Property Set resource. 
+This resource configuration corresponds to 'propertySets' config block in system configuration XML 
+(REST endpoint: artifactory/api/system/configuration).
+
+## Example Usage
+
+```hcl
+resource "artifactory_property_set" "foo" {
+  name 		= "property-set1"
+  visible 	= true
+
+  property {
+    name = "set1property1"
+
+    predefined_value {
+      name 			= "passed-QA"
+      default_value 	= true
+    }
+
+    predefined_value {
+      name 			= "failed-QA"
+      default_value 	= false
+    }
+
+    closed_predefined_values 	= true
```

Comment:
Let's make sure HCL is formatted nicely.

Reply:
done